openedx/course-discovery
# feat: add organizationMapping model for external org codes

### Overview

This PR covers the following JIRA ticket:
[PROD-3199](https://2u-internal.atlassian.net/browse/PROD-3199)

This PR covers the creation of OrganizationMapping model to map the `external_org_codes` to internal `Organization` model.

### Testing
- Run migrations and verify the model is created
    ```bash
    ./manage.py migrate course_metadata
    ```
- Verify the model is created in the admin panel
- Run the tests
```bash
pytest course_discovery/apps/course_metadata/tests/test_models.py::OrganizationMappingTests --ds=course_discovery.settings.test_local
```
